.github/workflows/etl-pipeline-tests.yaml

```diff
@@ -39,13 +46,6 @@ jobs:
             etl-pipeline/requirements.txt
             etl-pipeline/dev-requirements.txt
 
-      - name: Install build essentials
-        run: |
-          sudo apt-get update
-          sudo apt-get install -y gcc-12 g++-12
-          sudo update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-12 60
-          sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-12 60
-
```

Comment:
I wonder why moving the placement of this code improves the test time.

Reply:
I'm assuming because the c packages that are installed here weren't getting cached (since this was happening after the pip install step which checks the cache.